docs/contribute/packages/documentation.rst

```diff
@@ -0,0 +1,55 @@
+Documentation Guidelines
+------------------------
+
+Pyfar follows the :doc:`numpy style guide<numpydoc:format>` for the docstring. A docstring has to consist at least of
+
+- a short and/or extended summary,
+- the Parameters section, and
+- the Returns section.
+
+Optional fields that are often used are
+
+- References,
+- Examples, and
+- Notes.
+
+Here are a few tips to make things run smoothly.
+
+- Use the tags ``:py:func:``, ``:py:mod:``, and ``:py:class:`` to reference functions, modules, and classes from the package itself: For example ``:py:func:`~pyfar.plot.time``` for a link that displays only the function name. For links with custom text use ``:py:mod:`plot functions <pyfar.plot>```.
```

Comment:
```suggestion
- Use the tags ``:py:func:``, ``:py:mod:``, and ``:py:class:`` to reference functions, modules, and classes from the package itself. For example ``:py:func:`~pyfar.plot.time``` for a link that displays only the function name, for links with custom text use ``:py:mod:`plot functions <pyfar.plot>```.
```